dom.bs

```diff
@@ -1306,43 +1306,47 @@ for discussion).
      <li><p>Set <var>slot-in-closed-tree</var> to false.
     </ol>
 
-   <li><p>Let <var>clearTargetsTuple</var> be the last tuple in <var>event</var>'s
+   <li><p>Let <var>clearTargetsStruct</var> be the last struct in <var>event</var>'s
    <a for=Event>path</a> whose <a for=Event/path>target</a> is non-null.
 
-   <li><p>Let <var>clearTargets</var> be true if <var>clearTargetsTuple</var>'s
-   <a for=Event/path>target</a>, <var>clearTargetsTuple</var>'s <a for=Event/path>relatedTarget</a>,
-   or an {{EventTarget}} object in <var>clearTargetsTuple</var>'s
-   <a for=Event/path>touch target list</a> is a <a for=/>node</a> and its <a for=tree>root</a> is a
-   <a for=/>shadow root</a>, and false otherwise.
-
-   <li><p>Set <var>event</var>'s {{Event/eventPhase}} attribute to {{Event/CAPTURING_PHASE}}.
```

Comment:
As far as I can tell "legacy-pre-activation behavior" cannot trigger side effects so setting this later is not observable.

Reply:
Actually, it looks like we fire change event on radio buttons & checkboxes so the event phase change is observable. Gecko follows the current spec and WebKit & Blink uses the new behavior.

https://gist.github.com/rniwa/6c502dca3e16d5816db7958ce7bab4d7

Reply:
Thanks, we should add that test to the test suite. None seems like a reasonable phase to me given that traversal hasn't started yet, WDYT @smaug----?

Reply:
Agreed that None seems more reasonable than capturing given the event dispatching hasn't even started yet.

Reply:
https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=6198

Reply:
From API user's point of view Gecko's behavior makes more sense, since for them the event dispatch clearly has started already once dispatchEvent is called.

Reply:
I disagree. Just because we're in `dispatchEvent`, doesn't mean `eventPhase` needs to be something other than "none".